package/kedro_viz/launchers/cli.py

```diff
@@ -80,8 +80,11 @@ def commands():  # pylint: disable=missing-function-docstring
     help=PARAMS_ARG_HELP,
     callback=_split_params,
 )
+# pylint: disable=import-outside-toplevel
 def viz(host, port, browser, load_file, save_file, pipeline, env, autoreload, params):
     """Visualise a Kedro pipeline using Kedro viz."""
+    from kedro_viz import server
```

Comment:
Is there a specific reason to import server module instead of
`from kedro_viz.server import run_server`?

Reply:
Just to fulfill requirements of linter: there are two functions imported from this module: `run_server` and `is_localhost`. In case of importing them separately, linter tests report error that there are more than 15 local variables (16). Importing whole `server` fulfills this requirement and doesn't require to bypass checks.

I'm pretty convinced that in case of performance it doesn't make a big difference. In case of code style - I leave this decision to maintaners. My the only one intent was to pass linter tests without bypassing them.

Reply:
I am not sure why this increases the number of variables, as I understand the changes are moved 2 of them to `kedro.constants` and moving some of the import from the top level to the function lazy imports. So the total number of local imports should be unchanged.

I would prefer to keep the top-level module and suppress the listings in this case.

Reply:
> I am not sure why this increases the number of variables, as I understand the changes are moved 2 of them to `kedro.constants` and moving some of the import from the top level to the function lazy imports. So the total number of local imports should be unchanged.

When you import names inside the function, they are being put in `locals()` dictionary, as opposed to top-level imports when names are being put into `globals()`.

Number of local imports differ between `from kedro_viz.server import is_localhost, run_server` and `from kedro_viz import server` because in the first case you import two names (`is_localhost` and `run_server`) and in the second case only one (`server`) is imported.

> I would prefer to keep the top-level module and suppress the listings in this case.

What exactly do you mean by keeping the top-level module?

Reply:
Thank you for explaining! I thought it is complaining about the imported objects instead of the function locals.

I mean this keeping this  `from kedro_viz.server import run_server`

Reply:
I'll change this in the evening

Reply:
Done